config/application.rb

```diff
@@ -23,5 +23,17 @@ class Application < Rails::Application
     # config.i18n.default_locale = :de
 
     config.react.addons = true
+    # Don't generate extra files when generating models and controllers
+    config.generators do |generate|
+      generate.helper               false
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.